pennylane/fermi/fermionic.py

```diff
@@ -303,6 +303,115 @@ def to_mat(self, n_orbitals=None):
 
         return mat
 
+    def commute(self, source, target):
+        r"""Apply anti-commuting relations until the operator in position source is in position target.
+
+        Args:
+            source (int): The position of the operator to be commuted.
```

Comment:
Please also consider using more descriptive names for "source" and "target". Not sure what are the best names but also not sure the current ones are the best.

Reply:
How about "initial_pos" and "final_pos"? I'm open to suggestions if that isn't good enough.